joomla/joomla-cms
# [com_config] Use type="number" fields for com_config servers port fields

#### Summary of Changes

Use type="number" for com_config servers port fields.
Also add default values, hints and a range from 1 to 65536.

Before
![image](https://cloud.githubusercontent.com/assets/9630530/17196083/ee455374-5459-11e6-8b21-2b7d055c6d08.png)

After
![image](https://cloud.githubusercontent.com/assets/9630530/17196116/269a26d2-545a-11e6-9b11-45a9f2fd11cd.png)
#### Testing Instructions
1. Apply patch
2. Check all the port fields in global config
3. Save config
4. Repeat step 2 and check all is fine
5. Code review. 

Note: I added some default values for ftp_port and proxy_port port fields (didn't have default), please check also if that causes no trouble. Also now when removing the port and saving it goes back to the default port.
